evidence.go

```diff
@@ -0,0 +1,279 @@
+package ccatoken
+
+import (
+	"crypto"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"strings"
+
+	cose "github.com/veraison/go-cose"
+	ptoken "github.com/veraison/psatoken"
+)
+
+// claimType to hold the value of claim
+
+type claimType int
```

Comment:
this looks like a misnomer?  it's not a claim but the whole evidence (realm or platform) we are trying to differentiate.  A more apt name could be `evidenceType` or maybe `claimsType` (plural), or (using EAT naming conventions) `claimSetType`

Reply:
I would prefer claimsType (plural) version. Done it this way and corrected the internal types to be plural as well!